api/api.py

```diff
@@ -0,0 +1,43 @@
+from urls import url
+from requesting import request_from
+#Defining a client class
+class Client():
```

Comment:
@vimalsheoran  Update the Class Name with Bolt because we have changed the library syntax.